src/Installation/guides/source.md

```diff
@@ -13,6 +13,19 @@ It is recommended to ensure that you have installed Wine 8.3 or above on your sy
 
 To build Vinegar from source, you will need to install Go; it can be found packaged as `golang` or `go` on some Linux distributions.
 
+## For Arch and derived:
+```sh
+sudo pacman -S go
+```
+## For Ubuntu and derived:
+```sh
+sudo apt install golang-go
```

Comment:
This wont work, Go for Ubuntu is out of date and Vinegar requires Go 1.20.

Reply:
so the only options is basically to compile a binary?

Reply:
Flatpak or use the Go PPA

Reply:
Which didn't work for some...

Reply:
> Which didn't work for some...

ubuntu is proving to be a hard operating system to use

Reply:
Remove the ubuntu instructions

Reply:
> Remove the ubuntu instructions

yes i will, and i will make a script for ubuntu users to use

Reply:
> Remove the ubuntu instructions

i will make a script ant then i will pr an update to the wiki

Reply:
No. Remove it.

Reply:
> No. Remove it.

i removed it if you dont see
![image](https://github.com/vinegarhq/vinegarhq.github.io/assets/95132822/4d16c6dd-e2c6-453d-a2ed-55e5c49da187)